### ISSUE TITLE
[HOLD for 0xmiroslav] [$1000] Web - Test 'Assignee` is cut of from the bottom

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23115]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on Floating Plus button.
2. Click on assign task.
3. Enter any title.
4. Click on Next

## Expected Result:
The assignee 'g' should not be cut-off

## Actual Result:
The assignee 'g' is cut off

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.41-2
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/6e9229a2-4dd1-4e4b-8ee6-49e3445eceaa

![Screenshot 2023-07-18 at 10 22 33](https://github.com/Expensify/App/assets/93399543/1645e53c-4652-4230-9c56-a42212ad089d)

https://github.com/Expensify/App/assets/93399543/a9c0a02b-afbc-4a94-84c4-42f8d51e2c57

**Expensify/Expensify Issue URL:**
**Issue reported by:** @esh-g
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689438298925379

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01aa67583e6a2f7981</li>
        <li>Upwork Job ID: 1681403142046572544</li>
        <li>Last Price Increase: 2023-07-18</li>
    </ul>
</details>

**Comment by ahmedGaber93** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
[DEV] Test 'Assignee` is cut of from the bottom
### What is the root cause of that problem?
The applied style has small line height `styles.lhNormal`.
https://github.com/Expensify/App/blob/6e5b07cf45944f28373e188d6f02620a12d6da29/src/components/MenuItem.js#L99

### What changes do you think we should make in order to solve the problem?
We need to increase the current line height `lhNormal`, I think `lineHeightLarge` is a good enough, but we can also increase it to `lineHeightXLarge` if we found any issues in PR.

1. Create new style `lhLarge` after `lhNormal`.
```javascript
    lhLarge: {
        lineHeight: variables.lineHeightLarge,
    },
```
2. Change style in `MenuItem` to be `lhLarge` instead of `lhNormal`
https://github.com/Expensify/App/blob/6e5b07cf45944f28373e188d6f02620a12d6da29/src/components/MenuItem.js#L99

### What alternative solutions did you explore? (Optional)
apply the new style to this `MenuItem` only
pass `lhLarge` style in `MenuItem` prop `descriptionTextStyle` in NewTaskPage

**Comment by DanutGavrus** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Task assignee is cut of from the bottom.

### What is the root cause of that problem?
We are using `styles.textLabelSupporting` & `styles.lhNormal` on our `descriptionTextStyle` which contains the *Assignee* text:
https://github.com/Expensify/App/blob/25c8ae5212f433ea7679ba4bd040c79f151a282a/src/components/MenuItem.js#L96-L103
`styles.textLabelSupporting`:
https://github.com/Expensify/App/blob/25c8ae5212f433ea7679ba4bd040c79f151a282a/src/styles/styles.js#L1052-L1056
`styles.lhNormal`:
https://github.com/Expensify/App/blob/25c8ae5212f433ea7679ba4bd040c79f151a282a/src/styles/styles.js#L1076-L1078

1. `textLabelSupporting` uses the `fontSizeLabel` which uses `getValueUsingPixelRatio(13, 19)`;
2. `lhNormal` uses the `lineHeightNormal` which uses `getValueUsingPixelRatio(16, 21)`;
3. By using these values, we may get a line height which is lower than the text size.

### What changes do you think we should make in order to solve the problem?
We should use a *higher* `lineHeight`. For example, we have 2 other *Text Label* styles in the app:
https://github.com/Expensify/App/blob/25c8ae5212f433ea7679ba4bd040c79f151a282a/src/styles/styles.js#L292-L302
Both are using `lineHeightLarge`, so we may use it here too. I would remove the `styles.lhNormal` from the styles array and add the corresponding *lineHeight* in the `textLabelSupporting` style as we have in other ones.

### What alternative solutions did you explore?
As an alternative, we may simply replace `lhNormal` from the styles array with a *new* style: `lhLarge` for example.

**Comment by kbecciv** [comment]
# Proposal

## Please re-state the problem
Assignee text is cut-off in Menu item of task.

## What is the root cause of this issue?
This was introduced in this PR: https://github.com/Expensify/App/pull/22484
Here we used to have a style `styles.lineHeightNormal` which seemed to be a non-existent style and had no effect. But in the above mentioned PR, this non-existent style was identified and replaced with `styles.lhNormal` which seems to have caused this.

## What changes do you think are needed in order to solve this problem?
We should just get rid of the style `styles.lhNormal` altogether because it will fix this problem and behave as it did before.

## What alternative approach did you explore?
We can also set the line height to something more than normal like `lh20` or `lh140Percent` which already exist.

**Comment by jasperhuangg** [comment]
Super minor display bug, don't think this needs to be a blocker, gonna remove the deploy blocker label

**Comment by pecanoro** [comment]
@jasperhuangg In that case, I will assign the external label!

**Comment by jeffmchang** [comment]
Was anyone else able to reproduce the bug? I'm looking on my dev server and it looks fine.

**Comment by pecanoro** [comment]
Sounds good, assigning @esh-g 

However, a reminder to avoid posting proposals until the External label is assigned, if @grgia would have been online or an actual deploy broker, we could have fixed it internally without any type of retribution.

**Comment by pecanoro** [comment]
@esh-g You need to leave a comment here since it's not letting me assign you!

**Comment by esh-g** [comment]
@0xmiroslav Thanks for your positive comment. @pecanoro I'm sorry but I think it is an recommended practice to post the proposal on slack when reporting the bug...

I will have the PR ready today!

**Comment by pecanoro** [comment]
@esh-g Oh I didn't mention it just because of your proposal but everyone that had proposed in this issue after you, but I guess the same could have happened with yours. We are usually ok with getting a contributor for deploy blockers but if they are urgent, it's likely we will bypass it.

**Comment by esh-g** [comment]
@0xmiroslav The PR is here: https://github.com/Expensify/App/pull/23156 🚀

**Comment by JmillsExpensify** [comment]
Summary of payments for when we get past the regression period

 - Issue reporter: @esh-g $250
 - Contributor: @esh-g $1,500 (incl. 50% bonus)
 - Contributor+: @0xmiroslav $1,500 (incl. 50% bonus)

Upwork job [is here](https://www.upwork.com/jobs/~01aa67583e6a2f7981).

**Comment by JmillsExpensify** [comment]
Jumping in to pay now.

**Comment by JmillsExpensify** [comment]
Extended contracts to both, will pay the bonus on final payout. @0xmiroslav Please don't forget the BZ checklist items.

**Comment by pecanoro** [comment]
@0xmiroslav friendly bump to fill out the list!

**Comment by pecanoro** [comment]
@JmillsExpensify All yours!

**Comment by pecanoro** [comment]
@JmillsExpensify friendly bump!

**Comment by pecanoro** [comment]
@JmillsExpensify pinged you in newdot in case you missed this issue!

**Comment by JmillsExpensify** [comment]
Apologies for the delay! @0xmiroslav I believe the Upwork contract is waiting on you to accept? It's showing a pending status on my end.

**Comment by JmillsExpensify** [comment]
Do you mind asking having them apply here: https://www.upwork.com/jobs/~01aa67583e6a2f7981. Upwork makes it hard to find this profile and hire them for this specific job.

**Comment by pecanoro** [comment]
Not overdue

**Comment by JmillsExpensify** [comment]
@0xmiroslav let us know when you're ready to receive payments. Going to put this on hold until then.

**Comment by pecanoro** [comment]
Still on HOLD

**Comment by JmillsExpensify** [comment]
@0xmiroslav similar to other issues, I'm going to close this issue since I'm overwhelmed with GH assignments. Please reach out to us when you are ready for payment.
